src/tests/suite/dfuse_test.c

```diff
@@ -176,6 +177,92 @@ do_ioctl(void **state)
 	assert_int_equal(dur.uid, geteuid());
 	assert_int_equal(dur.gid, getegid());
 
+	rc = unlinkat(root, "ioctl_file", 0);
+	assert_return_code(rc, errno);
+
+	rc = close(root);
+	assert_return_code(rc, errno);
+}
+
+static bool
+timespec_gt(struct timespec t1, struct timespec t2)
+{
+	if (t2.tv_sec == t1.tv_sec)
+		return t2.tv_nsec < t1.tv_nsec;
```

Comment:
given that it's timespec_gt would it be a bit clearer if it were

return t1.tv_nsec > t2.tv_nsec

Reply:
Agree. Fixed. Copy paste is evil...